visualc/VS2010/aescrypt2.vcxproj

```diff
@@ -95,7 +95,7 @@
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
       <PreprocessorDefinitions>WIN32;_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-      <AdditionalIncludeDirectories>../../include</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>../../include;../../3rdparty/everest/include/;../../3rdparty/everest/include/everest;../../3rdparty/everest/include/everest/vs2010;../../3rdparty/everest/include/everest/kremlib</AdditionalIncludeDirectories>

```

Comment:
Done! If everyone adds their special constraints to `scripts/generate_visualc_files.pl` this may become ugly quickly. Like I stated in my commit message, it may be a better idea to add new, contribution-specific scripts in `3rdparty`.